docfx_project/ziti/overview.md

```diff
@@ -5,9 +5,60 @@ uid: zitiOverview
+
+This model enables Ziti systems to provide access to multiple services while ensuring that clients only get access to those services to which they have been granted access.    
+
+In addition to requiring cert based authentication for clients, Ziti uses certificates to authorize communication between Ziti components. 
+
+### Dark Services and Routers
+There are various levels of accessibility a network service can have.
+
+1. Many network services are available to the world. The service then relies on authentication and authorization policies to prevent unwanted access. 
+1. Firewalls can be used to limit access to specific IP or ranges. This increases security at the cost of flexibility. Adding users can be complicated and users may not be able to easily switch devices or access the service remotely.
+1. Services can be put behind a VPN or made only accessible to an internal network, but there are some downsides to this approach.
+    1. If you can access the VPN or internal network for any reason, all services in that VPN become more vulnerable to you.
+    1. VPNs are not usually appropriate for external customers or users.
+    1. For end users, VPNs add an extra step that needs to be done each time they want to access the service.
+1. Services can be made dark, meaning they do not have any ports open for anyone to even try and connect to. 
+
+Making something dark can be done in a few ways, but the way it's generally handled in Ziti is that services reach out and establish one or more connections to the Ziti network fabric. Clients coming into the fabric can then reach the service through these connections after being authenticated and authorized. 
+
+Ziti routers, which make up the fabric, can also be dark. Routers locoated in private networks will usually be made dark. These routers will reach out of the private network to talk to the controller and to make connections to join the network fabric mesh. This allows the services and routers in your private networks to make only outbound connections, so no holes have to opened for inbound traffic.
+
+Services can be completely dark if they embed a Ziti SDK. If this is not possible a Ziti tunneler or proxy can be colocated with the service. The service then only needs to allow connections from the local machine or network, depending on how close you colocate the proxy to the service.   
+
+### End to End Encryption
+If you take advantage of Ziti's developer SDKs and embed Ziti in your client and server applications, your traffic can be configured to be seamlessly encrypted from the client application to server application. If you prefer to use tunnelers or proxy applications, the traffic can be ecnrypted for you from machine to machine or private network to private network. Various combinations of the above are also supported.
+
+End-to-end encryption means that even if systems between the client and server are compromised, your traffic cannot be intercepted or tampered with.
```

Comment:
minor technical correction: traffic can be intercepted but not decrypted or tampered